onecgiar-pr-server/src/api/ipsr/innovation-pathway/innovation-pathway-step-three.service.ts

```diff
@@ -167,10 +172,14 @@ export class InnovationPathwayStepThreeService {
     }
   }
 
+  private validData(data: any, data_id: number, valid: number[]) {
+    return valid.includes(parseInt(`${data_id}`)) ? data : null;
```

Comment:
@JuanGuzman-io if `valid` is a `number[]`, why are we making the `dataId` a `string` and then parsing it as a `number`? Isn't `dataId` a number too?

Reply:
@gcmartinezg in theory if it is ok. Anyway, it is something that I did not implement, I would have to validate it more thoroughly. For now this validation works correctly.